pkg/kds/context/context_test.go

```diff
@@ -518,7 +520,7 @@ var _ = Describe("Context", func() {
 						k8sSystemNamespace: "custom-namespace",
 					},
 					name:         "foo.custom-namespace",
-					expectedName: "foo.custom-namespace",
+					expectedName: "foo-xw46fx28d8666769.custom-namespace",
```

Comment:
Should this hash be shorter? Feels like this detail should have been ironed out in the MADR.